sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/_message_backcompat.py

```diff
@@ -49,16 +49,17 @@ def __init__(self, message, **kwargs):
         self.delivery_no = kwargs.get('delivery_no')
         self.delivery_tag = kwargs.get('delivery_tag') or None
         self.on_send_complete = None
-        self.properties = LegacyMessageProperties(self._message.properties)
+        self.properties = LegacyMessageProperties(self._message.properties) if self._message.properties else None
```

Comment:
This resulted in an error if self._message.properties was None, so updated this.